core/templates/pages/learner-dashboard-page/learner-dashboard-icons.component.ts

```diff
@@ -0,0 +1,166 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for showing learner dashboard icons.
+ */
+
+import { Component, OnInit, Input} from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+import constants from 'assets/constants';
+import { LearnerDashboardIdsBackendApiService } from
+  'domain/learner_dashboard/learner-dashboard-ids-backend-api.service';
+import { LearnerPlaylistBackendApiService } from
+  'domain/learner_dashboard/learner-playlist-backend-api.service';
+import { LearnerDashboardActivityIds } from
+  'domain/learner_dashboard/learner-dashboard-activity-ids.model';
+
+@Component({
+  selector: 'learner-dashboard-icons',
```

Comment:
For all migrated components, add "oppia-" as a prefix. i.e, oppia-learner-dashboard-icons (do this elsewhere in the PR)

Reply:
done!